src/lib/datadog.ts

```diff
@@ -0,0 +1,17 @@
+import ddTracer from "dd-trace"
+
+if (process.env.DD_APM_ENABLED) {
```

Comment:
Should this be removed from the various `config` modules now that it's used directly here?

Reply:
Updated 👍

Reply:
Again not a blocker, but same applies to the 2 other `DD_*` configs.